nbs/common.base_recurrent.ipynb

```diff
@@ -130,6 +130,10 @@
     "        self.train_trajectories = []\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.                self.loss.distribution=='Bernoulli'
</pre><p> </p><p>Is Bernoulli the only loss distribution for temporal classification?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/565/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>